addons/mail/static/src/models/thread/thread.js

```diff
@@ -1630,6 +1630,21 @@ function factory(dependencies) {
             }
         }
 
+        /**
+         * @private
+         */
+        _onChangeLastMessageSetAsSeen() {
+            // Message from mailing channel should not make a notification in
+            // Odoo for users with notification "Handled by Email".
+            if (
+                this.mass_mailing &&
+                this.env.session.notification_type === 'email' &&
+                this.lastMessage
+            ) {
+                this.markAsSeen(this.lastMessage);
+            }
+        }
```

Comment:
This could be a performance problem, because `markAsSeen()` makes a RPC and this is called in a compute method.

Compute method can be called multiple times during a single update cycle, and this compute method is called whenever the last message changes, so can happen quite often. And this is done for each mailing channel, so if you have, let's say, 10 or 100 mailing channels and people are quite active, this can turn into hundreds or thousands or RPC in a short time.

This is called at least once per mailing channel at page load. So that's already problematic with 10-100 RPCs at least if you have 10-100 mailing channels. Note that server load is multiplied by amount of user that have mailing channels, so you can see how this can overload server at some points...